src/PSRule/Pipeline/SourcePipeline.cs

```diff
@@ -352,10 +342,29 @@ private static SourceFile[] IncludeFile(string path, string helpPath)
             return new SourceFile[] { new SourceFile(path, null, GetSourceType(path), helpPath) };
         }
 
-        private static SourceFile[] IncludePath(string path, string helpPath, string moduleName)
+        private static SourceFile[] IncludePath(string path, string helpPath, string moduleName, bool excludeDefaultRulePath)
+        {
+            if (excludeDefaultRulePath)
+            {
+                var filteredFiles = FilterFiles(
+                    path,
+                    directoryFilter: dir => !dir.Contains(
+                        DefaultRulePath.TrimEnd(Path.AltDirectorySeparatorChar),
+                        StringComparison.OrdinalIgnoreCase),
+                    filePattern: "*.Rule.*");
+
+                return GetSourceFiles(filteredFiles, helpPath, moduleName);
+            }
+
+            var allFiles = System.IO.Directory.EnumerateFiles(path, "*.Rule.*", SearchOption.AllDirectories);
+
+            return GetSourceFiles(allFiles, helpPath, moduleName);
+        }
```

Comment:
Used this workaround to filter out `ps-rule/` to fix the bug where files are searched twice. The `excludeDefaultRulePath` param basically ensures that directory is not searched, since by default `.ps-rule/` is searched during `Build()`.

Reply:
Let me know if we need to change this behaviour.